docs/questions/query-builder/filters.md

```diff
@@ -1,8 +1,8 @@
 ---
-title: Filters
```

Comment:
The other articles in this section are "summarizing" and "joining", plus "filters" makes me think of dashboard filters